Calculator.Tests/FunctionTests.cs

```diff
@@ -1,24 +1,110 @@
+namespace Calculator.Tests;
 using NUnit.Framework;
 using static Calculator.Functions;
 
-namespace Calculator.Tests;
-
 [TestFixture]
 public class FunctionTests
 {
     // good tests are split into the Three A's: Arrange, Act, and Assert
     [Test]
-    public void AddReturnsCorrectAnswer()
+    [TestCase(5, 4, 9)]    // Big positive number first, Small positive number second
+    [TestCase(4, 6, 10)]  //Small positive number, Big positive Number
+    [TestCase(-10, 6, -4)] //Big Negative Number, Small number
+    [TestCase(-12, 24, 12)] //Small Negative Number, Big Positive Number
+    [TestCase(-13, -7, -20)] //Big Negative Number, Small Negative Number
+    public void AddReturnsCorrectAnswer(int a, int b, int expectedSum)
     {
-        // arrange
+        // Arrange done by mapping TestCase Variables to Method Parameters
+        /*
         int a = 1;
         int b = 2;
         int expectedSum = 3;
+        */
 
         // act
-        int actualSum = Add(a, b);
+        var actualSum = Add(a, b);
 
         // assert
         Assert.AreEqual(expectedSum, actualSum);
     }
+
+    /*Fuction Name: SubtractReturnsCorrectAnswer
+    **Parameters: N/A
+    **Return: void
+    */
+    [Test]
+    [TestCase(5, 4, 1)]      // Big positive number first, Small positive number second
+    [TestCase(4, 6, -2)]     //Small positive number, Big positive Number
+    [TestCase(-10, 6, -16)]  //Big Negative Number, Small number
+    [TestCase(-12, 24, -36)] //Small Negative Number, Big Positive Number
+    [TestCase(-13, -7, -6)]  //Big Negative Number, Small Negative Number
+    [TestCase(-4, -12, 8)]   //Small Negative Number, Big Negative Number
+    public void SubtractReturnsCorrectAnswer(int a, int b, int expectedDiff)
+    {
+        // Arrange done by mapping TestCase Variables to Method Parameters
+        /*
+        int a = TestCaseAttribute(a);
+        int b;
+        int expectedDiff = -1;
+        */
+
```

Comment:
we can remove these comment blocks now

Reply:
In this case (and all others) should I leave the //Arrange done by mapping TestCase Variables to Method Parameters? 

Just because on line 8 you have the comment // good tests are split into the Three A's: Arrange, Act, and Assert.

Reply:
Yeah that's a good point, we can leave that first comment